packages/api/package.json

```diff
@@ -21,6 +21,7 @@
 	},
 	"devDependencies": {
 		"@regen-network/ts-proto": "1.105.2-1",
+		"eslint": "^8.23.1",
```

Comment:
should this be in the top-level package.json instead?

Reply:
yep 👍 updated